bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/dto/ActionDTOMapper.java

```diff
@@ -1,11 +1,10 @@
 /**
- * Copyright (c) 2016 Markus Rathgeb
+ * Copyright (c) 1997, 2016 by ProSyst Software GmbH and others.
```

Comment:
I am fine using a neutral version, but I don't see any reason why ProSyst should be added to files I have created.